### PR TITLE
Feat(eos_cli_config_gen): Implement VRRP for ethernet interfaces

### PR DESCRIPTION
**Pull request by gmuloc**
## Change Summary

Copied the "new" model from VLAN interfaces and checked it against a device, added some min/max where relevant (as well in VLAN)

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

VRRP current model for ethernet_interfaces

## How to test

molecule test

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2 [hunk 1]
```diff
@@ -384,6 +384,49 @@
...
+| --------- | ------- | -------- | ---------------------- | --------| ---------------------- | ------------------------ | --------------- | ----------------- | --------------- |
+{%         for ethernet_interface in ethernet_interfaces_vrrp_details | arista.avd.natural_sort('name') %}
+{%             for vrid in ethernet_interface.vrrp_ids if vrid.id is arista.avd.defined %}
+{%                 set row_id = vrid.id %}
+{%                 set row_prio_level = vrid.priority_level | arista.avd.default('-') %}
+{%                 set row_ad_interval = vrid.advertisement.interval | arista.avd.default('-') %}
+{%                 set row_preempt = "Enabled" %}
+{%                 if vrid.preempt.enabled is arista.avd.defined(false) %}
+{%                     set row_preempt = "Disabled" %}
+{%                 endif %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2:408]: Prempt is enabled by default on EOS. We do not configure `no vrrp x preempt` even when set to false. And we don't configure `vrrp x preempt` if set to true (without delay).
Delay and general preempt are two different commands in EOS:
```eos
 !
 interface Ethernet1/1
    shutdown
+   no vrrp 1 preempt
+   vrrp 1 preempt delay minimum 1 reload 2
 !
 interface Ethernet1/2
    shutdown
```

Comment (gmuloc) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2:408]: that's the code from Vlan - should it be removed there as well?

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2:408]: I think we should implement the rendering of `no vrrp x preempt` for the `false` case on both VLAN and Ethernet interfaces

**Review by ClausHolbechArista — APPROVED** [review]
LGTM. Crossing fingers for CI to pass :)

**Review by emilarista — APPROVED** [review]
Tested, LGTM!

**Pull request merged**